test/test_codegen.py

```diff
@@ -113,53 +111,6 @@ def test_generate_mechfile(lang_module, mechname):
         print(code, file=mech_file)
 
 
-@pytest.mark.parametrize("mechname", ["uiuc.yaml", "sandiego.yaml",
-                                      "uiuc.yaml"])
-@pytest.mark.parametrize("usr_np", numpy_list)
-def test_get_rate_coefficients(mechname, usr_np):
-    """This function tests that pyrometheus-generated code
-    computes the rate coefficients matching Cantera
-    for given temperature and composition"""
-    sol = ct.Solution(f"mechs/{mechname}", "gas")
-    ptk_base_cls = pyro.codegen.python.get_thermochem_class(sol)
-    ptk = make_jax_pyro_class(ptk_base_cls, usr_np)
-
-    three_body_reactions = [(i, r) for i, r in enumerate(sol.reactions())
-                            if r.reaction_type == "three-body-Arrhenius"]
-
-    # Test temperatures
-    temp = np.linspace(500.0, 3000.0, 10)
-    for t in temp:
-        # Set new temperature in Cantera
-        sol.TP = t, ct.one_atm
-        # Concentrations
-        y = sol.Y
-        rho = sol.density
-        c = ptk.get_concentrations(rho, y)
-        # Get rate coefficients and compare
-        k_ct = sol.forward_rate_constants
-        k_pm = ptk.get_fwd_rate_coefficients(t, c)
-
-        # It seems like Cantera 3.0 has bumped the third-body efficiency
-        # factor from the rate coefficient to the rate of progress
-        for i, r in three_body_reactions:
-            eff = np.sum([c[sol.species_index(sp)]
-                          * r.third_body.efficiencies[sp]
-                          for sp in r.third_body.efficiencies])
-            eff += np.sum([c[sp] for sp in range(sol.n_species)
-                           if sol.species_name(sp) not in
-                           r.third_body.efficiencies])
-            k_ct[i] *= eff
-
-        print(k_ct)
-        print()
-        print(k_pm)
-        print()
-        print(np.abs((k_ct-k_pm)/k_ct))
-        assert np.linalg.norm((k_ct-k_pm)/k_ct, np.inf) < 1e-14
-    return
-
-
```

Comment:
Why is this test being deleted?

Reply:
We are testing the reactions in a second test down below. I don't see why this one is necessary.